### PR TITLE
[cmake] fix build after 8ec984f0fa5bd8e75026e5558870b96071e54288

### PR DESCRIPTION
**Pull request by wsnipex**
build is broken under certain conditions, e.g. ubuntu packaging

## Motivation and Context

fixes build

## Types of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by wsnipex** [comment]
jenkins build this please
@AlwinEsch ping

**Comment by AlwinEsch** [comment]
Hi @wsnipex, yes was a fault from me and has forgot to change one place. Sorry.

See https://github.com/xbmc/xbmc/blob/master/project/cmake/treedata/common/subdirs.txt#L8 about the addonCallbacks_Game

File: xbmc/addons/interfaces/Game/CMakeLists.txt [hunk 1]
```diff
@@ -5,5 +5,5 @@ set(HEADERS AddonCallbacksGame.h)
 core_add_library(api1AddonCallbacks_Game)
```
Comment (AlwinEsch) [inline xbmc/addons/interfaces/Game/CMakeLists.txt]: This must be changed to **addonCallbacks_Game**

**Comment by wsnipex** [comment]
fixed
jenkins build this please

**Pull request merged**